rasa/core/policies/rule_policy.py

```diff
@@ -34,6 +34,11 @@
     USER_INTENT_SESSION_START: ACTION_SESSION_START_NAME,
 }
 
+RULES = "rules"
+NEGATIVE_RULES = "negative_rules"
```

Comment:
negative rules seems unintuitive to me, how about
```suggestion
FORM_UNHAPPY_RULES = "negative_rules"
```